extension/tests/container.sh

```diff
@@ -8,7 +8,7 @@ PG_VERSION=$1
 TESTCMD=$2
 
 docker build -t pgmemento-test--$PG_VERSION - < ./Dockerfile-$PG_VERSION;
-CONTAINER=$(docker run -d -v $(pwd)/../..:/home/pgmemento pgmemento-test--$PG_VERSION);
+CONTAINER=$(docker run -d -v $(pwd)/../..:/home/pgmemento pgmemento-test--$PG_VERSION -e POSTGRES_PASSWORD=password);
```

Comment:
```suggestion
CONTAINER=$(docker run -d -v $(pwd)/../..:/home/pgmemento -e POSTGRES_PASSWORD=password pgmemento-test--$PG_VERSION);
```

The parameters after the image name would be interpreted as extra arguments for the command run within the container. In this case we want to pass this parameter to the docker itself, so it should go before the image name (whir is `pgmemento-test--$PG_VERSION` in our case)

Reply:
thx. also found this myself by now :smiley: 
lets see